.appveyor.yml

```diff
@@ -64,14 +64,14 @@ install:
 
   ##################################################
   # Install packages for docs.
-  - choco install -y doxygen.portable imagemagick.tool ghostscript
+  - choco install --timeout 600 -y doxygen.portable imagemagick.tool ghostscript || true
```

Comment:
Please add a comment explaining, as was done for DR

Reply:
Ok, do we need to open a new issue in the DrMemory repo or take issue number from the DynamoRIO issue tracker ? (how do I point to the DynamoRIO issue tracker? using DR's i#XXXX ?)

Reply:
You can see the convention in the existing sources: DRi#XXXX

Reply:
done